src/scss/components/optional/_order-card.scss

```diff
@@ -41,6 +41,20 @@ $previousOrder-hoverBoxShadow: 0 2px 8px 0 rgba(0, 0, 0, 0.16), -2px -2px 4px 0
         }
     }
 
+    .c-orderCard--noOpenRestaurants {
```

Comment:
Maybe we could rename this class to `c-orderCard--inactive` or something similar? Just to keep classes a little more generic 😃

Reply:
Yep – 100%.

It would be good to tie this into the `.c-orderCard-image` reference too – so have two options:

```scss
.c-orderCard--inactive {
   .c-orderCard-image {
     …
   }
}

// or could make it a state class so that:
.is-inactive {
	&.c-orderCard {
	   …
    }
    .c-orderCard-image {
      …
    }
}
```

Up to you which one you use really – just a matter of opinion over using a modifier or a state class.